soh/soh/Enhancements/randomizer/logic.cpp

```diff
@@ -580,6 +580,8 @@ namespace Rando {
             case RE_WHITE_WOLFOS:
             case RE_WALLMASTER:
                 return CanJumpslash() || CanUse(RG_FAIRY_BOW) || CanUse(RG_FAIRY_SLINGSHOT) || CanUse(RG_BOMBCHU_5) || CanUse(RG_DINS_FIRE) || (CanUse(RG_BOMB_BAG) && (CanUse(RG_NUTS) || CanUse(RG_HOOKSHOT) || CanUse(RG_BOOMERANG)));
+            case RE_GERUDO:
+                return CanUse(RG_KOKIRI_SWORD) || CanUse(RG_MASTER_SWORD) || CanUse(RG_BIGGORON_SWORD);
```

Comment:
Can gerudos not be hit by anything else? No bow, explosives, hammer?

Reply:
I've used bombchus vs them. Want to test hammer etc. But figured with this PR I'd just clean up the existing logic

Reply:
Got the essence of why I'd usually recommend to split off this kinda stuff in a separate PR hehe. I'd like to ask to double check regardless though so it can't be forgotten if it's not too much trouble. Otherwise leave a comment above it with `// RANDOTODO: Figure out if other weapons can also kill the gerudo guards` or something like that.

Reply:
aside: bongo ignores hammer